packages/angular-material/src/other/master-detail/master.ts

```diff
@@ -170,7 +171,7 @@ export class MasterListComponent extends JsonFormsArrayControl {
     const controlElement = uischema as ControlElement;
     this.propsPath = props.path;
     const detailUISchema =
-      controlElement.options.detail ||
+      controlElement.options?.detail ||
       findUISchema(
```

Comment:
Not directly related to this PR but I just noticed this issue here: `findUISchema` already checks the `options.detail`, so we don't need to do this manually. Especially as our API also allows the strings `DEFAULT`, `GENERATED` and `REGISTERED` as valid `detail` values ;)

We can just hand over the `controlElement` as the next parameter after `'VerticalLayout'`, i.e.

```ts
const detailUISchema = findUISchema(
  props.uischemas,
  schema,
  `${controlElement.scope}/items`,
  props.path,
  'VerticalLayout',
  controlElement,
);
```